calculator.js

```diff
@@ -1,2 +1,81 @@
-var calculator = {
+
+ //var num1 = prompt("please enter first numbers");
+
+ //var num2 = prompt("please enter second numbers");
+
+console.log("The function name are addition, subtract, multiply, devide,power, and sqrt")
+
+ //var calculator = function addition("a","b","return (a + b)");
+//var myFunction = function add(a, b) {return a * b};
+var calculator={
+  addition: function addition(a, b)
```

Comment:
You can just write...

```js
addition: function(a, b){ 
  // code goes here
}
```

No need to write out `addition` a second time since you've already taken care of that with the object key.